src/NuGet.Clients/VsExtension/VsCredentialProviderImporter.cs

```diff
@@ -84,29 +84,47 @@ public class VsCredentialProviderImporter
         /// <returns>An enumeration of plugin providers</returns>
         public ICredentialProvider GetProvider()
         {
-            this._initializer();
             ICredentialProvider result = null;
 
-            if (ImportedProvider != null)
+            try
             {
-                result = new VsCredentialProviderAdapter(ImportedProvider);
-            }
+                this._initializer();
```

Comment:
The code exists this way before this change.  I would prefer not to increase the changes as this was supposed to be targeted.  As this is a nit are you ok with won't fix?